app/src/main/java/com/techlads/composetv/details/ProductDetailsScreen.kt

```diff
@@ -0,0 +1,198 @@
+        Spacer(modifier = Modifier.width(280.dp))
+        Button(
+            modifier = Modifier,
+            shape = RoundedCornerShape(4.dp),
+            contentPadding = PaddingValues(vertical = 0.dp, horizontal = 16.dp),
+            onClick = {  }) {
+            Text(modifier = Modifier.padding(horizontal = 16.dp),
+                text = "Play")
+        }
+        Spacer(modifier = Modifier.size(16.dp))
+        Text(
+            color = Color.White,
+            textAlign = TextAlign.Center,
+            style = MaterialTheme.typography.titleMedium,
+            text = "Watch Trailer")
+    }
+}
+
+@Composable
+fun DetailsSection() {
+    Row(modifier = Modifier
+        .fillMaxSize()
+        .background(color = Color.Black),
+        verticalAlignment = Alignment.Top,
+        horizontalArrangement = Arrangement.Center
+    ) {
+        Spacer(modifier = Modifier.size(220.dp))
+        Column(
+            Modifier
+                .fillMaxWidth()
+                //.weight(.7f)
+                .padding(24.dp),
+            horizontalAlignment = Alignment.Start) {
+            Spacer(modifier = Modifier.height(16.dp))
+            Text(modifier = Modifier,
+                color = Color.White,
+                textAlign = TextAlign.Start,
+                style = MaterialTheme.typography.headlineLarge,
+                text = "Batman")
+            Spacer(modifier = Modifier.size(10.dp))
+            MovieInfoSection()
+            Spacer(modifier = Modifier.size(10.dp))
+            Text(modifier = Modifier,
+                color = Color.White,
+                textAlign = TextAlign.Start,
+                style = MaterialTheme.typography.bodyLarge,
+                text = "Batman ventures into Gotham City's underworld when a sadistic killer leaves behind a trail of cryptic clues. ")
+        }
+    }
+}
+
+@Composable
+fun MovieInfoSection() {
+    Row(verticalAlignment = Alignment.CenterVertically,
+    horizontalArrangement = Arrangement.Center) {
+        Text(modifier = Modifier,
+            color = Color.White,
+            textAlign = TextAlign.Start,
+            style = MaterialTheme.typography.bodyLarge,
+            text = "2022")
+        Spacer(modifier = Modifier.size(12.dp))
+        Text(modifier = Modifier,
+            color = Color.White,
+            textAlign = TextAlign.Start,
+            style = MaterialTheme.typography.bodyLarge,
+            text = "2h 56mm")
+        Spacer(modifier = Modifier.size(12.dp))
+        Rating(rating = "8.3")
+    }
+}
+
+@Composable
+fun Rating(rating: String) {
+    Row(verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center) {
+        Image(
+            modifier = Modifier.size(18.dp),
+            painter = painterResource(id = R.drawable.ic_star),
+            contentDescription = "search",
+        )
+        Spacer(modifier = Modifier.size(8.dp))
+        Text(modifier = Modifier,
+            color = Color.White,
```

Comment:
Please remove hardcoded text color, see the existing implementation for light text